Core/include/Acts/Seeding/SeedfinderConfig.hpp

```diff
@@ -19,9 +19,18 @@ struct SeedConfirmationRange {
   float zMinSeedConf;
   float zMaxSeedConf;
   float rMaxSeedConf;
-  size_t nTopSeedConf;
   size_t nTopForLargeR;
   size_t nTopForSmallR;
+
+  SeedConfirmationRange(float zMinSC = std::numeric_limits<float>::min(),
+                        float zMaxSC = std::numeric_limits<float>::max(),
+                        float rMaxSC = std::numeric_limits<float>::max(),
+                        size_t nTopLargeR = 0, size_t nTopSmallR = 0)
+      : zMinSeedConf(zMinSC),
+        zMaxSeedConf(zMaxSC),
+        rMaxSeedConf(rMaxSC),
+        nTopForLargeR(nTopLargeR),
+        nTopForSmallR(nTopSmallR) {}
```

Comment:
Is there a need for a constructor here? Can the defaults not be put directly in the struct member declarations?

Reply:
done, removed the constructor as you said